DESCRIPTION

```diff
@@ -42,7 +39,14 @@ Imports:
     readr,
     rstudioapi,
     tibble,
-    tidytext
+    tidytext,
+    bib2df,
+    purrr,
+    stringr,
+    fs,
+    usethis,
+    gender,
+    remotes
```

Comment:
How are we doing on number of dependencies?

Reply:
We have 19 at the moment. A couple of depencencies were added with the addition of the bibliography analysis functions.